src/UCTNode.cpp

```diff
@@ -168,7 +168,7 @@ void UCTNode::kill_superkos(const KoState& state) {
 
 float UCTNode::eval_state(GameState& state) {
     auto raw_netlist = Network::get_scored_moves(
-        &state, Network::Ensemble::RANDOM_ROTATION);
+        &state, Network::Ensemble::RANDOM_ROTATION, 0, true);
```

Comment:
An assert in `get_scored_moves()` expects `-1` as value for `rotation` if `RANDOM_ROTATION` is used.